deployment/manifests/default.pp

```diff
@@ -17,6 +17,7 @@
 $ui_app_root = "${django_root}/ui"
 $data_dir = '/data'
 $django_docker_engine_data_dir = "${data_dir}/django-docker-engine-data"
+$django_docker_engine_mem_limit_mb = 20
```

Comment:
It seems like we wouldn't be able to test any of the beefier tools with a local Refinery after this change